pcap/pcapio/error.go

```diff
@@ -0,0 +1,38 @@
+package pcapio
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/brimsec/zq/zqe"
+)
+
+type ErrInvalidPcap struct {
+	err error
+}
+
+func NewErrInvalidPcap(err error) error {
+	e := &ErrInvalidPcap{err: err}
+	return zqe.E(zqe.Invalid, e)
+}
+
+func errInvalid(text string) error {
```

Comment:
Nit: consider removing `errInvalid(...)` and use `errInvalidf(...)` everywhere.

Reply:
sure!